api/seat/Seat.js

```diff
@@ -6,8 +6,20 @@ const seatSchema = new Schema({
   hall: {
     type: Schema.Types.ObjectId,
     ref: 'CinemaHall',
+    required: true,
+  },
+  row: {
+    type: Number,
+    required: true,
+  },
+  column: {
+    type: Number,
+    required: true,
+  },
+  name: {
```

Comment:
Is this necessary?I think row and column are enough

Reply:
I found it useful for iterating and search purposes.